example.py

```diff
@@ -0,0 +1,166 @@
+# https://github.com/Xilinx/brevitas/blob/4617f7bd136e96fa21c7f76e3c7e2e37fe563837/src/brevitas/export/onnx/qonnx/function.py#L36C1-L47C19
+import torch
+import torch.onnx.ops
+# def symbolic(g, x, scale, zero_point, bit_width, narrow_range, signed, rounding_mode):
+#     ret = g.op(
+#         f'{DOMAIN_STRING}::Quant',
+#         x,
+#         scale,
+#         zero_point,
+#         bit_width,
+#         rounding_mode_s=rounding_mode,
+#         signed_i=int(signed),
+#         narrow_i=int(narrow_range))
+#     ret.setType(x.type())
+#     return ret
+# def forward(ctx, x, scale, zero_point, bit_width, narrow_range, signed, rounding_mode):
+#     float_to_int_impl = solve_float_to_int_impl_from_enum(rounding_mode)
+#     quant = IntQuant(
+#         float_to_int_impl=float_to_int_impl(),
+#         tensor_clamp_impl=TensorClamp(),
+#         input_view_impl=Identity(),  #TODO: Update this when QONNX support Groupwise export
+#         narrow_range=narrow_range,
+#         signed=signed)
+#     y = quant(scale, zero_point, bit_width, x)
+#     return y
+
+DOMAIN_STRING = "onnx.brevitas"
+
+
+class Quant(torch.nn.Module):
+    def forward(
+        self,
+        x,
+        scale,
+        zero_point,
+        bit_width,
+        narrow_range: bool,
+        signed: bool,
+        rounding_mode: str,
+    ):
+        if torch.onnx.is_in_onnx_export():
+            return torch.onnx.ops.onnx_symbolic(
```

Comment:
torch.onnx.ops.Quant?

Reply:
I did it this way to allow type checkers to function. If we generate ops dynamically the type checkers will fail, and the ide will not be able to provide useful documentation either.